react/src/components/card/SprkCard.js

```diff
@@ -1,55 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
-import SprkCardHighlightedHeader from './components/SprkCardHighlightedHeader/SprkCardHighlightedHeader';
-import SprkCardTeaser from './components/SprkCardTeaser/SprkCardTeaser';
 
 const SprkCard = (props) => {
-  const {
-    additionalClasses,
-    additionalContentClasses,
-    children,
-    highlightedHeaderConfig,
-    idString,
-    isStandout,
-    teaserConfig,
-    variant,
-  } = props;
-
-  const GetCardContent = () => {
-    if (variant === 'highlightedHeader') {
-      return (
-        <SprkCardHighlightedHeader
-          highlightedHeaderConfig={highlightedHeaderConfig}
-        />
-      );
-    }
-
-    if (variant === 'teaser') {
-      return <SprkCardTeaser teaserConfig={teaserConfig} />;
-    }
-
-    return (
-      <div
-        className={cx(
-          'sprk-c-Card__content sprk-o-Stack__item',
-          additionalContentClasses,
-        )}
-      >
-        {children}
-      </div>
-    );
-  };
+  const { additionalClasses, children, idString, isStandout, ...other } = props;
 
   return (
     <div
       className={cx('sprk-c-Card sprk-o-Stack', additionalClasses, {
```

Comment:
Leaving sprk-o-Stack here would imply that someone would only ever put stack items in a card.

I'm okay with if we feel it's a reasonable assumption to make--I'm just thinking about a situation where someone might want to put something in a Card that's not a Stack Item

Reply:
@RVMendoza, I had a similar line of thought as well. But, ultimately, the Card is a layout pattern that needs the flexbox layout system. If this is omitted, the card's layout wouldn't render correctly. So, it needs to be there and it's more of a requirement that all the items are flexbox items. Because of the display: flex setting, the items in the card don't technically need to be official StackItems. They will inherit the base flex properties regardless. So our bases are covered here and we'll leave this class on there.

Reply:
So no one can ever use Card like this even if they don't intend to use flex styles?
```
<SprkCard>
   <div class="sprk-c-Card__content">
      <h2>Hello</h2>
      <p>I love cats</p>
   </div>
</SprkCard>
```

This implementation would receive unnecessary Stack styles that the user might not immediately know about.
We'd be boxing them into a flex container even if they don't want/need it.

It feels like if we keep it this way, we'd need to communicate that Cards will be a flex container and there's no way to choose otherwise.

Reply:
They can absolutely use that code that you pasted. It does feel that you may be picking at the code here with saying that we have to communicate that cards are flex containers and that we are restricting them otherwise. As with our other components, any of our styles are overridable via additionalClasses (display: flex cascading throughout a component would be comparable to font styles). In addition, our alerts, inputs, accordions, dictionary, footer, tooltip parts, and many other components all have display: flex on them. Almost all of our components are flex containers but we don't explicitly describe that CSS property for each component.